tests/test_main.py

```diff
@@ -221,3 +221,22 @@ def test_env_failure():
     task = train_tasks[0]
     approach.solve(task, timeout=500)
     _run_testing(env, approach)
+
+    # Test plan_only_eval succeeds, even though simulation fails.
+    utils.reset_config({
+        "env": "cover",
+        "approach": "random_actions",
+        "timeout": 10,
+        "make_test_videos": False,
+        "cover_initial_holding_prob": 0.0,
+        "num_test_tasks": 1,
+        "plan_only_eval": True
+    })
+    env = _DummyCoverEnv()
+    train_tasks = env.get_train_tasks()
+    approach = create_approach("oracle", env.predicates, env.options,
+                               env.types, env.action_space, train_tasks)
+    assert not approach.is_learning_based
+    task = train_tasks[0]
+    approach.solve(task, timeout=500)
+    _run_testing(env, approach)
```

Comment:
your comment says "Test plan_only_eval succeeds, even though simulation fails.". but i don't see where you're actually checking that it succeeds. do you mean to do something like...
```
metrics = _run_testing(env, approach)
assert metrics["num_solved"] == 1
```
?